README_CODE.md

```diff
@@ -1,13 +1,18 @@
-Last Revised: Feb 16, 2025
+Last Revised: Feb 18, 2025
 
 # MushroomObserver
 
 <!-- Most links are defined here for DRYness and consistency -->
 [Intro]: https://mushroomobserver.org/info/intro
 [Ruby Documentation]: https://www.ruby-doc.org/core/
+<!-- Zenspider seems to be broken. Is there a replacement? -->
 [Ruby Quick Ref]: https://www.zenspider.com/Languages/Ruby/QuickRef.html
```

Comment:
@mo-nathan The Ruby Quick Ref link does not work for me. I get:

> This site can’t be reached
> www.zenspider.com refused to connect.

Is there a substitute which you like?

Reply:
Interesting.  Works for me.  However, https://www.zenspider.com/ruby/quickref.html appears to be more current.